Vagrantfile

```diff
@@ -64,10 +64,11 @@ Vagrant.configure("2") do |config|
   # Puppet, Chef, Ansible, Salt, and Docker are also available. Please see the
   # documentation for more information about their specific syntax and use.
   config.vm.provision "shell", inline: <<-SHELL
-     wget https://packages.erlang-solutions.com/erlang-solutions_1.0_all.deb
+     wget --continue https://packages.erlang-solutions.com/erlang-solutions_1.0_all.deb
```

Comment:
Not sure this flag is necessary. What's the point?

Reply:
When re-provisioning the machine - it doesn't perform a full download request to packages.erlang-solutions.com - it sends a request for data greater than the currently downloaded file length. If that length is the same as the size of the file on the server nothing is downloaded.